docs/retrieve.txt

```diff
@@ -0,0 +1,248 @@
+.. _laravel-retrieve:
+
+==============
+Retrieve Data
+==============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find one, find many, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collections using **find operations**. Find operations are commands that
+retrieve documents from the server.
+
+This guide includes the following sections:
+
+- :ref:`laravel-find-setup`
+- :ref:`laravel-find-all`
+- :ref:`laravel-find-one`
+- :ref:`laravel-modify-find`
+- :ref:`laravel-addtl-info`
+
+.. _laravel-find-setup:
+
+Prerequisites
+-------------
+
+.. TODO: link to Quick Start once it's published
+
+The examples in this guide use the ``movies`` collection in the ``sample_mflix`` database,
+which is included in the MongoDB Atlas sample dataset. For instructions on loading the sample
+dataset into your database, see the Quick Start or the :atlas:`Get Started with Atlas Guide
+</getting-started/#atlas-getting-started>`.
+
+Before running the examples in this guide, you must create the following classes:
+
+- Model class to represent the ``movies`` collection
+- Controller class to store and run the find operations
+- View class to display the results of the find operations
+
+For instructions on creating these classes, see the Quick Start.
+
+.. _laravel-find-all:
+
+Find All Documents
+------------------
+
+To retrieve all documents in a collection, use the ``all()`` method.
+
+To retrieve all documents that match a set of criteria, use the ``where()`` method.
+Pass the field name and value for which you are querying as parameters to ``where()``.
+
+You can also pass a comparison operator as the ``where()`` method's second parameter,
+such as the ``>`` operator. If you do not include a comparison operator parameter,
+the find operation queries for documents whose field values are equal to the specified
+value.
+
+To apply multiple sets of criteria to the find operation, you can chain a series
+of ``where()`` methods together.
+
+Find All Documents in a Collection Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example retrieves all documents in the ``movies`` collection:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+
+      $movies = Movie::all();
```

Comment:
Q: does this code example also print the retrieved documents?

Reply:
The code snippet itself doesn't, but editing the view class according to the Quick Start instructions outputs the results